templates/profile_client/index.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello {{ user.firstname }}!{% endblock %}
+
+{% block body %}
+<main class="container-fluid col-11 profile">
+
+    <h1>Mon espace client</h1>
+
+    <section class="mes-informations border border border-dark container-fluid col-11">
+        <div class="d-flex align-items-center border border-dark title bg-light col-md-8 col-lg-6">
+            <h2 class="m-2">Mes informations</h2>
+            <i class="bi bi-pencil-square"></i>
+        </div>
+
+        <div class="bg-light my-2 px-3 py-2">
+            <p>Nom</p>
+            <h3>{{ user.lastname }}</h3>
```

Comment:
tu peux plutot faire direct un app.user.lastname ici, sans avoir besoin de passer $this->getUser() de ton controlleur à ta vue